core/services/postgres/q.go

```diff
@@ -174,5 +183,12 @@ func (q Q) GetNamed(sql string, dest interface{}, arg interface{}) error {
 	}
 	ctx, cancel := q.Context()
 	defer cancel()
+	q.logSql(query, args...)
 	return errors.Wrap(q.GetContext(ctx, dest, query, args...), "error in get query")
 }
+
+func (q Q) logSql(query string, args ...interface{}) {
+	if q.lggr != nil {
```

Comment:
I think `lggr` should probably be required so that we don't miss anything by mistake.

Reply:
Agreed, require logger for `postgres.NewQ` and then you can remove it from the Transaction call

Reply:
Alright, will make it required, but I will need to fix all NewQ usages then. Will leave it for the final iteration.